custom/icds_reports/views.py

```diff
@@ -2164,3 +2170,51 @@ def get(self, request, *args, **kwargs):
         except Exception:
             response = dict(isSuccess=False, message='Unknown Error occured')
             return JsonResponse(response, status=500)
+
+
+@location_safe
+@method_decorator([api_auth, toggles.ICDS_GOVERNANCE_DASHABOARD_API.required_decorator()], name='dispatch')
+class GovernanceAPIView(View):
+
+    @staticmethod
+    def get_state_id_from_state_code(state_code):
+        return AwcLocation.objects.get(aggregation_level=AggregationLevels.STATE,
+                                       state_site_code=state_code, state_is_test=0).state_id
+
+    def get_gov_api_params(self, request, *args, **kwargs):
+        step = kwargs.get('step')
+        now = datetime.utcnow()
+        month = int(request.GET.get('month', now.month))
+        year = int(request.GET.get('year', now.year))
+        state_id = request.GET.get('state_site_code')
```

Comment:
its actually site code . `state_site_code` would be better var name ehre.

Reply:
done